nbstripout/_utils.py

```diff
@@ -29,17 +29,19 @@ def pop_recursive(d, key, default=None):
     return default
 
 
-def _cells(nb, conditional=None):
```

Comment:
Given this is only called from a single place, maybe drop the default value, as it's always passed.